### PR TITLE
Make it possible to validate RDF data in Turtle format

### PR DESCRIPTION
**Pull request by Windvis**

### CODE REVIEW

File: config/environment.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ module.exports = function (environment) {
       // Here you can pass flags/options to your application instance
       // when it is created
     },
+    validationEndpoint: process.env.VALIDATION_ENDPOINT,
```
Comment (Windvis) [inline config/environment.js]: This doesn't use the Semantic Works stack yet, which is why I added it this way. It depends on a `.env` file in the root of the project with the correct endpoint url.

Comment (anitacaron) [reply config/environment.js]: So, do you want to update it before merging?

Comment (Windvis) [reply config/environment.js]: I need to make changes to the dispatcher first (which I'm _trying_ to do now :smile:). But I don't see it as a blocker for merging. I can swap out the config later.

Comment (Windvis) [reply config/environment.js]: Updated the dispatcher, so now it depends on this pr: https://github.com/lblod/app-poc-shacl-validation/pull/5

File: app/controllers/index.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    const response = yield fetch('/validate', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'text/turtle',
+      },
+      body: this.rdfData,
+    });
+
+    const { id: validationReportId } = yield response.json();
```
Comment (Windvis) [inline app/controllers/index.js:44]: This part depends on: https://github.com/lblod/app-poc-shacl-validation/pull/6

**Review by anitacaron — APPROVED** [review]

**Pull request merged**